app/services/station_service.rb

```diff
@@ -0,0 +1,19 @@
+class StationService
+
+  def self.get_nearest_station(address)
+    street_address = address.split(",")[0]
+    city = address.split(",")[1]
+    state = address.split(",")[2].split(" ")[0]
+    zipcode = address.split(",")[2].split(" ")[1]
+
+    response = conn.get("/api/alt-fuel-stations/v1.json?street_address=#{street_address}&city=#{city}&state=#{state}&zip=#{zipcode}&fuel_type=ELEC&limit=1&api_key=#{ENV['fuel_station_api_key']}")
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  private
+
+  def self.conn
+    Faraday.new(url: "https://developer.nrel.gov") 
+  end
+
+end
```

Comment:
THis all looks good. One thing you could do to better align with rails convention is to use the block syntax when assigning parameters in a request. https://lostisland.github.io/faraday/usage/#:~:text=response%20%3D%20conn.post(%27/post%27)%20do%20%7Creq%7C%0A%20%20req.params%5B%27limit%27%5D%20%3D%20100%0A%20%20req.body%20%3D%20%7Bquery%3A%20%27chunky%20bacon%27%7D.to_json%0Aend